site/content/docs/alerting-and-retries/retries.md

```diff
@@ -7,61 +7,107 @@ menu:
 cli: true
 ---
 
-Sometimes the internet is flaky, or your app is just having a hiccup not worth pinging your on-call team about. Retries 
-are your first line of defense against these types of false positives, leading to alert fatigue.
+Sometimes the internet is flaky, or your app is just having a hiccup not worth pinging your on-call team about. Retries are your first line of defense against these types of false positives, leading to alert fatigue.
+
+Whether you need to retry a check or a test session, there are different retries available.
 
 {{<warning>}}
-The "double check" option is deprecated and will be removed in the future. Please use the retries instead.
+Use **Retries** instead of the deprecated "double check" option.
 {{</warning>}}
 
+## Check retry strategies
 
-## Retry strategies
-
-Each check (except for Heartbeat checks) and each check group has a **Retries** section that allows you to configure how many times
-and according to which strategy a check should be retried.
+All check types and check groups, except for Heartbeat checks, have a **Retries & Alerting** section under **Edit > Check settings**, where you can define the number of retries and which retry strategy your check or check group uses.
 
 ![configure retry strategy](/docs/images/alerting/retries.png)
 
-You can pick from three distinct retry strategies that each come with different retry/backoff characteristics.
+### Check retries: on a schedule
+
+There are three distinct retry strategies based on the time between retry attempts, and each one comes with different retry/backoff characteristics.
 
 - **Fixed** — a fixed time between retries, e.g. 5s, 5s, 5s etc.
 - **Linear** — a linearly increasing time between retries, e.g. 5s, 10s, 15s etc.
 - **Exponential** — an exponentially increasing time between retries, e.g. 5s, 25s, 125s (2m and 5s)  etc. 
 
-You can also set a **maximum total retry duration** that controls the maximum time a check can be in a retrying state. 
-This is mostly a stop-gap for not chalking up too many retries.
+On top of the time between attempts, you can also set: 
 
-{{<info >}}
-Note that the **maximum total retry duration** also includes the time it takes for a check to run. So if you set the maximum to 2 minutes,
-your check takes 1.5 minutes, you have 30 seconds left for retries.
-{{</info >}}
+* **maximum number of retries** - the maximum number of retries for this check or check group. 
+* **maximum total retry duration** - the maximum time a check can be in a retrying state. 
+This is a timeout to ensure the check finishes on a timely manner.
+    
+    {{<info >}}
+    Make sure to include the time your check needs to run when setting the **maximum total retry duration**. For example, if you set a maximum to 2 minutes, and your check takes 1.5 minutes, you have 30 seconds left for retries.
```

Comment:
This info box looks weird in the preview - not sure if it's just me, or for everyone?

![CleanShot 2024-07-31 at 19 06 22](https://github.com/user-attachments/assets/a3435d64-6e13-46b8-be10-af360d52cfce)

Reply:
thanks for the catch! it's likely the indentation, I missed that locally

Reply:
<img width="719" alt="image" src="https://github.com/user-attachments/assets/762e4bb5-9a7c-4f67-899c-c7e04bdba99e">
Should be fixed!